config/schema/document_types/edition.json

```diff
@@ -29,6 +29,8 @@
     "slug",
     "specialist_sectors",
     "statistics_announcement_state",
-    "world_locations"
+    "world_locations",
+    "start_date",
+    "end_date"
```

Comment:
Have made that change.

Not sure if anything else is needed on this PR - I tried writing a simple integration test to check the field is being used but I'm either misunderstanding what the test's doing or I've forgotten something in the config because it's not showing up yet.